doc/user/content/releases/v0.50.md

```diff
@@ -1,13 +1,29 @@
 ---
 title: "Materialize v0.50"
 date: 2023-04-12
-released: false
+released: true
+patch: 3
 ---
 
-{{< warning >}}
-This version of Materialize is not yet released.
-{{< /warning >}}
-
 ## v0.50.0
 
-* No documented changes yet.
+#### SQL
+
+* Add [`mz_internal.mz_dataflow_arrangement_sizes`](/sql/system-catalog/mz_internal/#mz_dataflow_arrangement_sizes)
+  to the system catalog. This view describes how many records and batches are
+  contained in operators under each dataflow, which is useful to approximate how
+  much memory a dataflow is using.
+
+#### Bug fixes and other improvements
+
+* Improve the usability of subscriptions when using the [`WITH (PROGRESS)`](/sql/subscribe/#progress)
+  option. Progress information is now guaranteed to include a progress message
+  as the **first** update, indicating the `AS OF` time of the subscription.
+  This helps distinguish between an empty snapshot and an "in-flight" snapshot.
+
+* Improve the reliability of SSH tunnel connections when used with large cluster
+  sizes, and add more verbose logging to make it easier to debug SSH connection
+  errors during source and sink creation.
```

Comment:
Not a big deal, but the more verbose logging is internal only and so does not have a user-facing impact!

Reply:
I meant to encompass error messages in "logging", instead of spelling both out.

Reply:
i.e. #18583